### PR TITLE
fix: shop name alignment fixed

### PR DESCRIPTION
**Pull request by djsreeraj**

### CODE REVIEW

File: src/Pages/ChatView/PlaceOrder/index.tsx [hunk 1]
```diff
@@ -139,6 +140,10 @@ export default function PlaceOrder() {
 
     function handleSubmit() {
         setIsSubmit(true)
+        isValidCart && setPaymentModalVisible(true)
```
Comment (kirananto) [inline src/Pages/ChatView/PlaceOrder/index.tsx:143]: Why this approach ? Why not check it before submit ?

**Review by kirananto — CHANGES_REQUESTED** [review]

File: src/Pages/ChatView/PlaceOrder/index.tsx (hunk 1 above)
Comment (djsreeraj) [reply src/Pages/ChatView/PlaceOrder/index.tsx:143]: this submit is just to proceed to the modal, final submit is inside the modal code

Comment (kirananto) [reply src/Pages/ChatView/PlaceOrder/index.tsx:143]: Why is it not displayed by default ? Let it be displayed by default. 
Why modal ? Not inline in the page ?

Comment (djsreeraj) [reply src/Pages/ChatView/PlaceOrder/index.tsx:143]: not clear bro, need to talk on it

Comment (kirananto) [reply src/Pages/ChatView/PlaceOrder/index.tsx:143]: Let's talk more on this today

Comment (djsreeraj) [reply src/Pages/ChatView/PlaceOrder/index.tsx:143]: sure

Comment (kirananto) [reply src/Pages/ChatView/PlaceOrder/index.tsx:143]: Nvm this also works. we can merge this approach

**Pull request merged**